src/components/list/List.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ListItem from '../list/ListItem';
+
+const List = ({items, spaced, small}) => {
+  const listClass = classNames('sg-list', {
+    'sg-list--spaced-elements': spaced
+  });
+
+  return <ul className={listClass}>
+    {items.map(({text}, index) =>
```

Comment:
as I understand do we pass in items array of objects with 1 property text?
`items=[{text:'abc'},{text:'xyz'}]`
If yes, we should pass only directly text property, not opaqued in object